scripts/coverage.fish

```diff
@@ -1,52 +0,0 @@
-#!/usr/bin/env fish
-set -l c 0
-
-cd $INNERWORKDIR
-and rm -rf combined
-and mkdir combined
-and python3 "$WORKSPACE/jenkins/helper/aggregate_coverage.py" $INNERWORKDIR/gcov $INNERWORKDIR/combined/result
-
-and echo "creating gcno tar"
-and pushd ArangoDB/build
-and tar c -f /tmp/gcno.tar (find . -name "*.gcno")
-and popd
-and echo "copying gcno files"
-and tar x -f /tmp/gcno.tar -C $INNERWORKDIR/combined/result
```

Comment:
Should we as well create these tars in python script?

Reply:
the tar command here was used as a copy, it could have been `tar -cO heredir | tar -xC subdir -` - but it wasn't.
We use regular python copying for that purpose now.